portfolio/src/main/webapp/index.html

```diff
@@ -116,6 +116,137 @@ <h4 class="title">Bachelor of Eng. : Computer Science (2018 - Present)</h4>
 
     </section>
 
+    <!-- Work Experience -->
+    <section id="work-experience" class="work-experience bg-light">
+        <div class="container">
+
+            <h1>WORK EXPERIENCE</h1>
+
+            <div class="grid-3 center">
+                
+                <!-- Work experience 1 -->
+                <div>
+                    <h3>VEBITS AI</h3>
+                    <h4>Computer Vision & Deep Learning Analyst</h4>
+                    <p class="date">Jun 2019 - Aug 2019</p>
+                    <p>Driver Behaviour Detection</p>
+                    <ul class="description">
+                        <li>Enhanced a model that uses computer vision to identify a driver using the phone in real time.</li>
+                        <li>Refined TensorFlow's Object Detection API, & pretrained architectures like Faster RCNN and SSD to train the model, with high accuracy of 90%.</li>
+                        <li>Actively used OpenCV extensively for data visualization and model evaluation.</li>
+                    </ul>
+                </div>
+
+                <!-- Work experience 2 -->
+                <div>
+                    <h3>URECA RESEARCH - NTU</h3>
+                    <h4>Sentiment analysis & NLP Researcher</h4>
+                    <p class="date">Aug 2019 - Present</p>
+                    <p>Financial Events Analysis</p>
+                    <ul class="description">
+                        <li>Invited to prestigious President’s Undergraduate Research Program offered to top 10% of cohort.</li>
+                        <li>Developed a new deep learning model for stock price prediction, using Sentiment Analysis & Natural Language Processing with high accuracy of 85%.</li>
+                    </ul>
+                </div>
+
+                <!-- Work experience 2 -->
+                <div>
+                    <h3>VANTAGE WEBTECH</h3>
+                    <br>
+                    <h4>Android application Developer</h4>
```

Comment:
ditto, I think heading tags already start next elemnt at next line. Or consider using styling element instead.

Reply:
Oops, okay I'll add it to a styling element.